web/src/scripts/containers/TabbedEditor.jsx

```diff
@@ -125,6 +127,46 @@ class TabbedEditor extends Component {
     })
   }
 
+  renderFlowInstallationToast() {
+    const linkStyle = {
+      textDecoration: 'underline',
+    }
+
+    return (
+      <EditorToast
+        type={'recommended'}
+        buttonText={'Yes, install flow-bin into node_modules'}
+        message={(
+          <div>
+            Use the Flow static type-checker{' '}
+            <a
+              style={linkStyle}
+              onClick={() => shell.openExternal("https://flowtype.org/")}
+            >
+              (https://flowtype.org/)
+            </a>
+            {' '}for enhanced autocompletion and prop detection? Many Deco features
+            rely on Flow for static analysis of your project's code,
+            so this is highly recommended.
+          </div>
+        )}
+        onClose={() => {
+          this.props.dispatch(setFlowError(null))
+        }}
+        onButtonClick={() => {
+          const {rootPath, npmRegistry} = this.props
+
+          this.props.dispatch(setFlowError(null))
+
+          FlowController.getFlowConfigVersion()
```

Comment:
Put this in an action? This is the one thing I didn't want to happen with the remote requires.